src/cameras/src/ImageSaver.cpp

```diff
@@ -0,0 +1,96 @@
+/*
+ * Created By: Rowan Zawadzki
+ * Created On: Aug 8 2022
+ * Description: A node that saves images from the realsense cameras CIRC2022
+ */
+
+#include <ImageSaver.h>
+
+ImageSaver::ImageSaver(int argc, char **argv, std::string node_name) {
+  
+    // Setup NodeHandles
+    ros::init(argc, argv, node_name);
+    ros::NodeHandle nh;
+    ros::NodeHandle private_nh("~");
+    image_transport::ImageTransport it(nh);
+    //decide which camera to save from
+//std::string parameter1    = "camera";
+   // SB_getParam(private_nh, parameter1, camera, 2);
+    // Setup Subscriber(s)
+    std::string cam1_topic;
+   std::string cam2_topic;
+    cam1_topic = "cam_1/color/image_raw";
+   cam2_topic = "cam_2/color/image_raw";
+
+       std::string shutter1_topic = "shutter1_button";
+      std::string shutter2_topic = "shutter2_button";
+
+
+    
+    int queue_size                    = 10;
+    camera1_subscribe                  = it.subscribe(cam1_topic, queue_size, &ImageSaver::subscriberCallBack1, this);
+        camera2_subscribe                  = it.subscribe(cam2_topic, queue_size, &ImageSaver::subscriberCallBack2, this);
+
+    int queue_size2                    = 4;
+    shutter1                  = nh.subscribe(shutter1_topic, queue_size2, &ImageSaver::subscriberCallBackShutter1, this);
+    shutter2                  = nh.subscribe(shutter2_topic, queue_size2, &ImageSaver::subscriberCallBackShutter2, this);
```

Comment:
Not urgent given competition is around but it may be helpful to use a more descriptive name for the callbacks